src/main/java/cloud/eppo/BanditEvaluationResult.java

```diff
@@ -0,0 +1,73 @@
+package cloud.eppo;
+
+import cloud.eppo.ufc.dto.DiscriminableAttributes;
+
+public class BanditEvaluationResult {
+
+  private final String flagKey;
+  private final String subjectKey;
+  private final DiscriminableAttributes subjectAttributes;
```

Comment:
`DiscriminableAttributes` is an interface I introduced for attributes that can broken up (discriminated) by numeric and contextual. You'll see it more in action later, but basically, it helps abstract converting broken-out attributes to combined and vice versa.